accumulator/bittricks.md

```diff
@@ -0,0 +1,34 @@
+# Bit tricks
+
+There are various bitwise operations that can be used to compute positions of relatives of a node in the forest.
+
+**Example tree with 8 leaves (15 nodes) and forestRows=3:**
+```
+         decimal:                          binary:
+row 0:   14                                1110
+         |---------------\                 |-----------------------\
+row 1:   12              13                1100                    1101
+         |-------\       |-------\         |-----------\           |-----------\
+row 2:   08      09      10      11        1000        1001        1010        1011
+         |---\   |---\   |---\   |---\     |-----\     |-----\     |-----\     |-----\
+row 3:   00  01  02  03  04  05  06  07    0000  0001  0010  0011  0100  0101  0110  0111
+```
+
+## Placement
+The placement (left, right) of a node's position is indicated by its least significant bit (LSB 0). 0/1 = left/right. The LSB 1 indicates the parent's placement, the LSB 2 indicates the placement of the grandparent, ..., LSB n indicates the placement of the nth ancestor but only if n is smaller than the row index of the node.
```

Comment:
nit here but ```0/1 = left/right``` looked like division in a first glance. Maybe a different symbol would explain it better?